README.md

```diff
@@ -40,18 +40,10 @@ but all configuration options of [OpenTelemetry Instrumentation for Java] are av
 
 ## Getting Started
 
-### Configure your application
+> **Important**: Please use the [Grafana Agent](#grafana-agent) configuration for production use cases.
 
-You can use the [Grafana Agent](#grafana-agent) or the
-[Grafana Cloud OTLP Gateway] to send telemetry data to Grafana Cloud.
-
-#### Grafana Cloud OTLP Gateway
-
-> **Important**: Please use the Grafana Agent configuration for production use cases.
-
-The easiest setup is to use the Grafana Cloud OTLP Gateway, because you don't need to run any service to transport the
-telemetry data to Grafana Cloud.
-The Grafana Cloud OTLP Gateway is a managed service that is available in all Grafana Cloud plans.
+The easiest setup is Cloud OTLP Gateway - no additional service required
```

Comment:
It could be beneficial to describe why use this setup and what makes it easy? For example:

For an easier local development setup skip an exporter (Grafana Agent or OpenTelemetry Collector) and send directly to Grafana Cloud OTLP Gateway.